### PR TITLE
Feature/student training data

### PR DESCRIPTION
**Pull request by arman-tdev**

### CODE REVIEW

File: v1.0/frontend/SaralApp/src/modules/ScanHistory/ScanHistory.js [hunk 1]
```diff
@@ -28,10 +28,6 @@ const ScanHistory = ({
 
 
     useEffect(() => {
-        setIsLoading(true)
-        setTimeout(() => {
-            setIsLoading(false)
```
Comment (dileep-gadiraju) [inline v1.0/frontend/SaralApp/src/modules/ScanHistory/ScanHistory.js:33]: @arman-tdev , why is this changed? please merge properly

**Comment by arman-tdev** [comment]
1) loading issue when press on save all scan it should be there
2) trainingdata if student and marks predicted properly so we will not save that student id's trainingData either we will save that student id and marks

**Comment by apurvabisht97** [comment]
I have tested above pr . I am getting traiing data only for those sheets where student id and marks are not scanned properly . 
And now load is also shown after scan when we try to save our marks to db . now we cannot click save all scan again and again while saving it to db .

**Review by dileep-gadiraju — APPROVED** [review]
